Sources/Sentry/include/SentryTransport.h

```diff
@@ -6,14 +6,21 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSInteger, SentryTransportFlushResult) {
+    kSentryTransportFlushResultSuccess = 0,
+    kSentryTransportFlushResultTimedOut,
+    kSentryTransportFlushResultAlreadyFlushingBeforeLock,
+    kSentryTransportFlushResultAlreadyFlushingInLock
+};
```

Comment:
l: Im not a huge fan of this over-engineering just to be able to test it.
This values are not used anywhere in the project other than tests, and `kSentryTransportFlushResultAlreadyFlushingInLock` is never used.